api/spec/v1.yaml

```diff
@@ -276,13 +242,13 @@ paths:
         - in: query
           name: minFee
           schema:
-            <<: *BigIntType
+            allOf: [$ref: '#/components/schemas/BigInt']
```

Comment:
Can this be named anything else, e.g. `StringifiedBigInt`
It is misleading in typescript, where BigInt is already a different global type

Reply:
Hm doesn't typescript do namespacing quite well?
Anyway, renamed to `TextBigInt` to keep the name a bit shorter.

Reply:
Our code generator handles it correctly as long as we don't have any native BigInts in that file; so it would only be misleading when looking at the code and intellisense